README.md

```diff
@@ -105,6 +105,13 @@ Rollbar.configure({enabled: false});
 Rollbar.error("This will *not* be reported to Rollbar");
 ```
 
+## Dealing with adblocker / browser extension exceptions
```

Comment:
Comment regarding placement in this file: can you move it closer to the end, just before the "ios9 webviews" section?